modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,771 @@
+For more information on local and remote database aliases as part of a composite database, see xref::database-administration/composite-databases/manage-composite-databases.adoc#create-composite-database-alias[Create database aliases in composite databases].
+
+The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+
+[NOTE]
+====
+Database alias names are subject to the rules specified in the xref:database-administration/aliases/naming-aliases.adoc[Alias names] section.
+====
+
+[[alias-management-create-local-database-alias]]
+=== Create database aliases for local databases
+
+A local database alias targets a database within the same DBMS.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2021`
+----
+
+When you create a local database alias, it shows up in the `aliases` column provided by the command `SHOW DATABASES` and in the `SHOW ALIASES FOR DATABASE` command.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE `northwind`
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                   | type       | aliases       | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "northwind-graph-2021" | "standard" | ["northwind"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `northwind` FOR DATABASE
+----
+
+.Result
+[role="queryresult]
+----
++---------------------------------------------------------------------------+
+| name        | composite | database               | location | url  | user |
++---------------------------------------------------------------------------+
+| "northwind" | NULL      | "northwind-graph-2021" | "local"  | NULL | NULL |
++---------------------------------------------------------------------------+
+----
+ 
+=== Use `IF EXISTS` or `OR REPLACE` when creating database aliases
+
+The `CREATE ALIAS` command is optionally idempotent, with the default behavior to fail with an error if the database alias already exists.
```

Comment:
since this section is in the middle of the local alias ones, should it explicitly say it works for both local and remote aliases?

Reply:
(I know we have a comment about it in the create remote section so maybe not needed?)

Reply:
One argument for it is that both IF NOT EXISTS and OR REPLACE will check for both local and remote aliases so you trying to create a local if not exists when a remote exists will do nothing

Reply:
Will leave for now.